### PR TITLE
Run accumulo checks at WARN, and fix some easier spots

### PR DESCRIPTION
**Pull request by alerman**

### CODE REVIEW

**Review by apmoriarty — DISMISSED** [review]

File: warehouse/core/src/main/java/datawave/mr/bulk/BulkInputFormat.java [hunk 1]
```diff
@@ -54,8 +56,6 @@
 import org.apache.accumulo.core.security.TablePermission;
 import org.apache.accumulo.core.singletons.SingletonReservation;
 import org.apache.accumulo.core.util.Pair;
```
Comment (ddanielr) [inline warehouse/core/src/main/java/datawave/mr/bulk/BulkInputFormat.java:58]: Pair can be replaced with [AbstractMap.SimpleEntry](https://docs.oracle.com/javase/8/docs/api/java/util/AbstractMap.SimpleEntry.html) at a later date.

File: checkstyle.xml [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
-    <module name="RegexpSinglelineJava">
-<!--      <property name="format" value="import\s+org\.apache\.accumulo\.(.*\.(impl|thrift|crypto)\..*|(?!core|minicluster|testing).*|core\.(?!client|conf|data|security|spi|iterators).*)"/>-->
-      <property name="format" value="import\s+org\.apache\.accumulo\.(?!(core\.(client|data|iterators|security)|minicluster|hadoop)\.).*" />
-      <property name="ignoreComments" value="true" />
-      <property name="message" value="Accumulo non-public classes imported" />
+    <!--check that only Accumulo 2.1.x public APIs are imported-->
+    <module name="ImportControl">
+        <!--Allows per-package enforcement for importing Accumulo 2.1.x public APIs -->
+        <property name="file" value="import-control-accumulo.xml"/>
+        <property name="severity" value="warning"/>
```
Comment (ddanielr) [inline checkstyle.xml:12]: With this change, the build now shows these messages for disallowed imports
```
[WARN] /data/workspace/datawave/warehouse/query-core/src/main/java/datawave/core/iterators/ColumnRangeIterator.java:19:1: Disallowed import - org.apache.accumulo.core.iteratorsImpl.system.InterruptibleIterator. [ImportControl]
```

Verified that switching this severity line to `error` fails the build on ImportControl violations.

**Review by ddanielr — DISMISSED** [review]
Looks good! 
The build prints out the disallowed imports without failing so they can be worked off incrementally without causing disruption.

**Comment by ddanielr** [comment]
One thing to consider. This change fails if dropping down to a sub-directory to build via `mvn org.apache.maven.plugins:maven-checkstyle-plugin:3.3.0:check` and the documented `mvn -Pdocker,dist -DskipMicroservices clean install -T 1C` build command. 

Build failure due to checkstyle not finding the `import-control-accumulo.xml` file.  
Solution could be to add additional files per sub-directory, or figure out how to inject a root project path in for the file location. 

Approved since the README does not direct users to build from the sub-directories.

**Comment by ddanielr** [comment]
> One thing to consider. This change fails if dropping down to a sub-directory to build via `mvn org.apache.maven.plugins:maven-checkstyle-plugin:3.3.0:check` and the documented `mvn -Pdocker,dist -DskipMicroservices clean install -T 1C` build command.
> 
> Build failure due to checkstyle not finding the `import-control-accumulo.xml` file. Solution could be to add additional files per sub-directory, or figure out how to inject a root project path in for the file location.
> 
> Approved since the README does not direct users to build from the sub-directories.

Added a commit that fixes this problem

**Review by ddanielr — APPROVED** [review]

**Review by ivakegg — APPROVED** [review]

**Pull request merged**